packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -212,25 +213,37 @@ const DefaultSummaryConfiguration: ISummaryConfiguration = {
     maxAckWaitTime: 120000,
 };
 
-/**
- * Options for container runtime.
- */
-export interface IContainerRuntimeOptions {
-    // Flag that will generate summaries if connected to a service that supports them.
-    // This defaults to true and must be explicitly set to false to disable.
-    generateSummaries?: boolean;
-
-    // Delay before first attempt to spawn summarizing container
-    initialSummarizerDelayMs?: number;
-
-    // Flag that will disable garbage collection if set to true.
+export interface IGCRuntimeOptions {
+    /* Flag that will disable garbage collection if set to true. */
     disableGC?: boolean;
 
-    // Flag that will bypass optimizations and generate GC data for all nodes irrespective of whether the node
-    // changed or not.
+    /**
+     * Flag representing the summary's preference for enabling garbage collection.
```

Comment:
nit: This comment need a small update to reflect the new name.

Reply:
Fixed